Server/mods/deathmatch/logic/CPed.h

```diff
@@ -278,6 +305,9 @@ class CPed : public CElement
     std::vector<CPlayer*>::const_iterator NearPlayersIterBegin() { return m_nearPlayersList.begin(); }
     std::vector<CPlayer*>::const_iterator NearPlayersIterEnd() { return m_nearPlayersList.end(); }
 
+    const SPlayerAnimData& GetAnimationData() const noexcept { return m_animData; };
+    void                   SetAnimationData(const SPlayerAnimData& animData) noexcept { m_animData = animData; };
```

Comment:
One more thing. This method can potentially throw bad_alloc exception. I would not mark it as noexcept.